crates/turborepo-lockfiles/src/npm.rs

```diff
@@ -135,6 +138,16 @@ impl Lockfile for NpmLockfile {
     fn encode(&self) -> Result<Vec<u8>, crate::Error> {
         Ok(serde_json::to_vec_pretty(&self)?)
     }
+
+    fn global_change_key(&self) -> Vec<u8> {
```

Comment:
This is something that I'm not super passionate about, but I think for readability it might make more sense to use something like `serde_json::Value` here as it's slightly more ergonomic to construct.
e.g.
```
json!({
    "type": "npm",
    "version": self.lockfile_version,
    "requires": self.other.get("requires").cloned().unwrap_or(serde_json::Value::Null)
})
```

Reply:
I also considered pulling in bincode which may be a little faster. I know readability is not great. I don't really have a great intuition for how hot these functions are but I am thinking it's called only a handful of times during a run so I am going to just use he macro.

Reply:
This will only get called at most once during a run and only if the git based filter is used and there's a lockfile change across those commits i.e. it's not hot at all.